### DISCUSSION TITLE
Player / Squads bug after being revived many times.

### DISCUSSION THREAD

**Discussion by P4nz3rW4r** [Xeno69/Domination#468] [Q&A]
Hello Xeno, my group and I have been constantly going through a very annoying bug, it always happens when players die and are revived several times and that's when the bug happens, players seem to change from blufor team to opfor or something appeared because they are no longer able to enter vehicles with the rest of the team and also their names no longer appear on the map and even in the squad manager. The only way to resolve the bug is for the affected player to disconnect from the server and reconnect.

Any idea where the problem could be?

**Comment by Xeno69** [comment]
Nope, because I play on a server that is often maxed out (40 players) and have never encountered the problem.

Anyway, which version do you play and what addons and/or mods do you use?

Is a RPT from a client where it happens and a server RPT when it happens available?

**Comment by P4nz3rW4r** [comment]
Tonight we will play and the bug will certainly occur again with the players, as soon as it occurs I will note the time of the event and bring the RPT file for checking.

The problem occurs in both version 4.62 and the latest 4.64. As for MODs, well we used some... And many Arma 2 ports like CUP, RHS and etc...


<html>
<body>


Blastcore Edited (standalone version) | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=767380317
-- | -- | --
CBA_A3 | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=450814997
CUP Terrains - Core | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=583496184
CUP Terrains - Maps | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=583544987
CUP Units | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=497661914
CUP Vehicles | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=541888371
CUP Weapons | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=497660133
Enhanced Movement | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=333310405
Extended fire (for Blastcore Edited) | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=857496480
F/A-18E/F Super Hornet 2020 | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=2131302796
Helicopter Dust Efx Mod | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=1537745369
JSRS SOUNDMOD | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=861133494
RHSAFRF | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=843425103
RHSGREF | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=843593391
RHSSAF | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=843632231
RHSUSAF | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=843577117
Tier One Weapons | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=2268351256
USAF Mod - Fighters | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=2397371875
USAF Mod - Main | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=2397360831
Vanilla smoke (for Blastcore Edited) | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=853303947
VSM All-In-One Collection | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=823636749
Zeus Enhanced | Steam | https://steamcommunity.com/sharedfiles/filedetails/?id=1779063631


</body>
</html>

**Comment by P4nz3rW4r** [comment]
[RPT Files.zip](https://github.com/Xeno69/Domination/files/10009021/RPT.Files.zip)

I stopped being seen on the map and in squad management right after being revived by another player, wrote down the exact time it was revived and went to check the RPT log and found these two error lines:

```
0:40:08 Duplicate weapon Throw detected for B_T_Soldier_Repair_F
0:40:08 Duplicate weapon Put detected for B_T_Soldier_Repair_F
```

It looks like it really has something to do with the Respawn system.
I found this topic by searching google.
[https://feedback.bistudio.com/T165130](url)

We use AI recruitment enabled on our server. I will disable AI recruitment to test whether the bug stops or continues.

**Comment by Xeno69** [comment]
There is just a small error in the server RPT file which has nothing to do with respawn with revive (add points) but no ther errors.

What you have is quite a lot of object not messages from the game. Maybe adjust your server networking params.
And another thing you could try is turning remote exec safety in description.ext (CfgRemoteExec mode = 0)

Other than that you are using some which may cause the issue.
